pontoon/pretranslation/tests/test_pretranslate.py

```diff
@@ -735,38 +730,77 @@ def test_get_pretranslations_fluent_complex(
     pretranslated_string = serializer.serialize_entry(parser.parse_entry(expected))
 
     response = get_pretranslation(fluent_entity, google_translate_locale)
-    assert response == (pretranslated_string, tm_user)
+    assert response == (pretranslated_string, "tm")
+
+
+@patch("pontoon.pretranslation.pretranslate.get_google_translate_data")
+@pytest.mark.django_db
+def test_get_pretranslations_fluent_placeholders(
+    gt_mock, fluent_resource, google_translate_locale
+):
```

Comment:
This test is the only one of the `test_transformer.py` tests that doesn't replicate other tests already in this file.